ethcore/src/executive.rs

```diff
@@ -257,46 +267,45 @@ impl<'a> Executive<'a> {
 				}
 			}
 		} else {
-			// if destination is a contract, do normal message call
-			
-			// don't trace if it's DELEGATECALL or CALLCODE.
```

Comment:
In latest commit I've also added comments next to relevant if statements.

https://github.com/ethcore/parity/pull/903/commits/4bfbb56701b48115bdb1cf797b974fbe73a777f8